partials/loop.hbs

```diff
@@ -36,6 +36,9 @@
           <h2 class="m-article-card__title js-article-card-title {{#unless feature_image}}js-article-card-title-no-image{{/unless}}" title="{{title}}">
             {{title}}
           </h2>
+          {{#if meta_description}}
```

Comment:
Same comment as above, also note that might be better to use `excerpt` instead of `meta_description`

Reply:
Thanks for the `excerpt` tip